articles/data-explorer/ingest-data-event-grid.md

```diff
@@ -147,13 +147,28 @@ Save the data into a file and upload it with this script:
     az storage container create --name $container_name
 
     echo "Uploading the file..."
-    az storage blob upload --container-name $container_name --file $file_to_upload --name $blob_name
+    az storage blob upload --container-name $container_name --file $file_to_upload --name $blob_name --metadata "rawSizeBytes=1024"
 
     echo "Listing the blobs..."
     az storage blob list --container-name $container_name --output table
 
     echo "Done"
 ```
+> [!NOTE]
+> In order to achieve the best ingestion performance, it is important to let Kusto know the **uncompressed** size of the compressed blobs submitted for ingestion.
+> Since Event Grid notifications only contain the basic details, the size information needs to be communicated explicitly.
+> Uncompressed size information can be provided by setting the `rawSizeBytes` property on the blob metadata to **uncompressed** data size in bytes.
+
+### Ingestion Properties
+You can specify [Ingestion properties](https://docs.microsoft.com/en-us/azure/kusto/management/data-ingestion/#ingestion-properties) of the blob ingestion via the blob metadata.
+These follow properties can be set:
+    * `rawSizeBytes` - Size of the raw data, i.e., uncompressed. For Avro/ORC/Parquet this should be the size before the format-specific compression is applied.
+    * `kustoTable` -  Name of the existing target table. Will override the `Table` set on the connection.
+    * `kustoDataFormat` -  Data format. Will override the `Data format` set on the connection.
```

Comment:
The format is specified in the command, not in the connection

Reply:
It can be specified both in the data connection & metadata, I'll add explanation.